docs/edgeql/functions/converters.rst

```diff
@@ -23,3 +23,23 @@ the functions below allow more options for such conversions.
 
         db> SELECT to_json('{"hello": "world"}')['hello'];
         {'world'}
+
+.. eql:function:: std::to_str(json: json, fmt: OPTIONAL str={}) -> str
+
+    :index: stringify dumps
+
+    Return string representation of the input value.
+
+    When converting :eql:type:`json`, this function can take
+    ``'prety'`` as the optional *fmt* argument to produce
+    pretty-formatted JSON string.
+
+    This is the reverse of :eql:func:`to_json`.
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT to_str(<json>2);
+        {'2'}
+
+        db> SELECT to_str(<json>['hello', 'world']);
+        {'["hello", "world"]'}
```

Comment:
I'd also add an example of pretty output.

Reply:
It looks *ugly* in repl:
```
views> SELECT to_str(<json>(a := 2, b := 'hello'), 'pretty');
{'{\n    "a": 2,\n    "b": "hello"\n}'}
```

Reply:
We'll fix that.

Update to

```
views> SELECT to_str(<json>(a := 2, b := 'hello'), 'pretty');
{'{
    "a": 2,
    "b": "hello"
}'}
```